index.html

```diff
@@ -412,7 +412,7 @@
             The list of valid post properties is defined by the vocabularies. This allows new vocabularies to be developed outside the development of the syntax.
           </p>
           <p>
-            Most values will be strings. If a property (such as `author` for example) references another
+            Most values will be strings. If a property (such as <a>author</a> for example) references another
```

Comment:
@dissolve I think so too. I should've mentioned though - a lot of the properties come from mf2 instead of directly in the spec, so there's no matching `<dfn>` when you link them with Respec - we could either format those separately or just manually link to the mf2 spec?